data/blog/8th-wave-of-bitcoin-grants.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Eighth Wave of Bitcoin Grants"
+date: '2024-10-23'
+tags: ['OpenSats', 'grants', 'bitcoin', 'wave']
+draft: false
+authors: ['arvin', 'dergigi']
+images: ['/static/images/blog/59-bitcoin-8th-wave.jpg']
+summary: "OpenSats is funding 4 more open-source projects in the Bitcoin ecosystem."
+---
+
+OpenSats is proud to announce its eighth wave of Bitcoin grants, dedicated to
+supporting innovative open-source projects that enhance the growth and
+resilience of the Bitcoin ecosystem.
+
+We remain committed to empowering developers worldwide who are building
+groundbreaking tools, improving user experience, and strengthening the overall
+security of Bitcoin and its users.
+
+This wave awards grants to four outstanding projects, each playing a vital role
+in advancing various aspects of the Bitcoin network. In no particular order, the
+recipients are:
+
+- [Citadel-Tech](#citadel-tech)
+
+- [Lampo](#lampo)
+
+- [Cashu Nutshell](#cashu-nutshell)
+
+- [PickhardtPayments Plugin](#pickhardtpayments-plugin)
+
+These grants are sourced from our General Fund, sustained by the generosity of
```

Comment:
Consider saying that the General Funds are exclusively spent on open-source projects more directly in the text. I think that is an amazingly unique aspect of Opensats, so people should hear it as much as possible (but briefly). Consider the following text: 

> These grants are sourced from our General Fund, which directly supports open-source projects. Sustained by donors who share our vision for a thriving, decentralized future, 100% of General Fund donations go to Bitcoin-related projects, not operational costs. To support these projects and help Bitcoin flourish, please consider donating to the General Fund: